pkg/controller/replication/replication_controller_test.go

```diff
@@ -1,1719 +0,0 @@
-/*
-Copyright 2014 The Kubernetes Authors.
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-// If you make changes to this file, you should also make the corresponding change in ReplicaSet.
-
-package replication
```

Comment:
Is every test covered by RS unit test? Is it safe to remove all RC unit tests?

Reply:
We discussed this offline. For the record:

The underlying RS code we're now using is unit tested in the RS package. The only question is whether the RS code interacts properly with the conversion layer, and that makes more sense as integration tests (which I've now added for RC) and e2e tests (which already exist for RC).